src/app/pages/storage/volumes/manager/manager.component.ts

```diff
@@ -126,6 +128,26 @@ export class ManagerComponent implements OnInit, AfterViewInit {
     stripe: 1, mirror: 2, raidz: 3, raidz2: 4, raidz3: 5,
   };
 
+  includeNonUniqueSerialDisks = false;
+  nonUniqueSerialDisks: ManagerDisk[] = [];
+  get availableNonUniqueSerialDisksCount(): number {
+    if (this.originalDisks.length === this.temp.length) {
+      return this.nonUniqueSerialDisks.length;
+    }
+    return this.temp.filter((disk) => this.nonUniqueSerialDisks.includes(disk)).length;
```

Comment:
We should rename `temp` to something more readable

Reply:
Your suggestions?